kuma/wiki/templates/wiki/includes/document_macros.html

```diff
@@ -216,7 +216,7 @@
     {% set short_share_url_gplus = share_url_gplus | bitly_shorten %}
 
     {% set twitter_url = 'http://twitter.com/share'|urlparams(url=short_share_url_twitter, text=share_text) %}
-    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + share_text %}
+    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=' + MEDIA_URL + 'img/favicon144.png&p[title]=' + share_text %}
```

Comment:
[This article](http://ar.zu.my/how-to-really-customize-the-deprecated-facebook-sharer-dot-php/) implies that full URLs are valid. Regardless, [Facebook says](https://developers.facebook.com/x/bugs/357750474364812/) that arguments are ignored.

That seems to be consistent with the behavior I'm seeing. The following link specifies google.com as the URL and our favicon as the image, but the Google logo is used as the image anyway.

http://facebook.com/sharer.php?s=100&p[url]=https://google.com&p[images][0]=https://developer.cdn.mozilla.net/media/img/favicon144.png&p[title]=I%20learned%20about%20Template:test%20on%20MDN.

I'm gonna go ahead and remove the image argument. cc @stephaniehobson